$NetBSD: patch-Cargo.toml,v 1.1 2025/02/26 00:49:17 schmonz Exp $

fix: revert rust 2024 edition
("we don't really use any 'features' of 2024 edition")
Upstream commit 888b2d23503e7dbb9323d40b6de0c3e6fa401292

--- Cargo.toml.orig	2025-02-26 00:43:10.386946933 +0000
+++ Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 name = "codeberg-cli"
 version = "0.4.9"
-edition = "2024"
+edition = "2021"
 license = "AGPL-3.0-or-later"
 keywords = ["codeberg-cli", "cli", "codeberg", "api", "tool"]
 authors = [
